ToggleItem/ToggleItem.js

```diff
@@ -26,11 +26,11 @@ import ForwardRef from '@enact/ui/ForwardRef';
 import Pure from '@enact/ui/internal/Pure';
 import PropTypes from 'prop-types';
 import Toggleable from '@enact/ui/Toggleable';
-import {ToggleItemBase as UiToggleItem} from '@enact/ui/ToggleItem';
 import Touchable from '@enact/ui/Touchable';
 import Spottable from '@enact/spotlight/Spottable';
 import compose from 'ramda/src/compose';
 
+import {ToggleItemBase as UiToggleItem} from '../internal/ToggleItem';
```

Comment:
please update the jsdoc in this file  line138

Reply:
Done